Specs/Data/Models/glTF-2.0/PropertyTextureWithTextureTransform/glTF/PropertyTextureWithTextureTransform.gltf

```diff
@@ -0,0 +1,197 @@
+    {
+      "buffer": 0,
+      "byteOffset": 0,
+      "byteLength": 12,
+      "target": 34963
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 12,
+      "byteLength": 128,
+      "byteStride": 32,
+      "target": 34962
+    }
+  ],
+  "samplers": [
+    {
+      "magFilter": 9728,
+      "minFilter": 9728,
+      "wrapS": 33071,
+      "wrapT": 33071
+    }
+  ],
+  "textures": [
+    {
+      "source": 0,
+      "sampler": 0
+    }
+  ],
+  "images": [
+    {
+      "uri": "PropertyTextureWithTextureTransform_img0.png"
+    }
+  ],
+  "buffers": [
+    {
+      "uri": "PropertyTextureWithTextureTransform_data.bin",
+      "byteLength": 140
+    }
+  ],
+  "materials": [
+    {
+      "doubleSided": true,
+      "pbrMetallicRoughness": {
+        "metallicFactor": 0,
+        "baseColorTexture": {
+          "index": 0,
+          "extensions": {
+            "KHR_texture_transform": {
+              "offset": [
+                0.25,
+                0.25
+              ],
+              "scale": [
+                0.5,
+                0.5
+              ]
+            }
+          }
+        }
+      }
+    }
+  ],
+  "meshes": [
+    {
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 1,
+            "NORMAL": 2,
+            "TEXCOORD_0": 3
+          },
+          "mode": 4,
+          "material": 0,
+          "indices": 0,
+          "extensions": {
+            "EXT_structural_metadata": {
+              "propertyTextures": [
+                0
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "nodes": [
+    {
+      "mesh": 0
+    }
+  ],
+  "scenes": [
+    {
+      "nodes": [
+        0
+      ]
+    }
+  ],
+  "scene": 0,
+  "extensionsUsed": [
+    "EXT_structural_metadata",
+    "KHR_texture_transform"
+  ],
+  "extensionsRequired": [
+    "KHR_texture_transform"
+  ],
+  "extensions": {
+    "EXT_structural_metadata": {
+      "schema": {
+        "id": "EXAMPLE_SCHEMA_ID",
+        "classes": {
+          "exampleClass": {
+            "name": "Example Class",
+            "properties": {
+              "exampleProperty": {
+                "name": "Example property",
+                "type": "SCALAR",
+                "componentType": "UINT8",
+                "normalized": true
+              }
+            }
+          }
+        }
+      },
+      "propertyTextures": [
+        {
+          "class": "exampleClass",
+          "properties": {
+            "exampleProperty": {
+              "index": 0,
+              "channels": [
+                0
+              ],
+              "extensions": {
+                "KHR_texture_transform": {
+                  "offset": [
+                    0.25,
+                    0.25
+                  ],
+                  "scale": [
+                    0.5,
+                    0.5
+                  ]
+                }
+              }
+            }
+          }
+        }
+      ]
+    }
+  }
+}
```

Comment:
missing newline at the end of file

Reply:
This is an "auto-generated" file (i.e. written by VSCode glTF plugin in this exact form), so one could make a case to not apply hand-edits. But it will eventually be maintained manually (and human-readable), so that's probably not a huge deal here.